### PR TITLE
Strip trailing whitespace from commands

### PR DESCRIPTION
**Pull request by LemonBoy**
We probably need to do that for some other commands too...

### CODE REVIEW

File: src/core/commands.c [hunk 1]
```diff
@@ -748,6 +748,11 @@ int cmd_get_params(const char *data, gpointer *free_me, int count, ...)
 			if (cnt == 0 && count & PARAM_FLAG_GETREST) {
 				/* get rest */
 				arg = datad;
+
+				/* strip the trailing whitespace */
+				if (count & PARAM_FLAG_STRIP_TRAILING_WS) {
+					arg = g_strchomp (arg);
```
Comment (dequis) [inline src/core/commands.c]: You should strip the trailing whitespace after the function name

**Comment by LemonBoy** [comment]
Updated to cover some more cases, review dis kthxbai

**Comment by ailin-nemui** [comment]
seems useful to me, any other opinions?

**Comment by dequis** [comment]
Yeah it seems good. The approach is very conservative so the only potential problem i can imagine is having to add this flag to other commands not considered here, and that is a good thing.

**Pull request merged**